nf_core/lint.py

```diff
@@ -237,9 +237,9 @@ def check_files_exist(self):
             os.path.join('docs','README.md'),
             os.path.join('docs','output.md'),
             os.path.join('docs','usage.md'),
-            ['.travis.yml', os.path.join('.github', 'workflows', 'branch.yml'), os.path.join('.circleci','config.yml')],
-            ['.travis.yml', os.path.join('.github', 'workflows','ci.yml'), os.path.join('.circleci','config.yml')],
-            ['.travis.yml', os.path.join('.github', 'workflows', 'linting.yml'), os.path.join('.circleci','config.yml')]
+            ['.travis.yml', os.path.join('.github', 'workflows', 'branch.yml')],
```

Comment:
I think this will make Maxime's linting fail again, cause he does not have .travis.yml or github actions workflows. Hence I added here the CircleCI, to also allow for this

Reply:
this checks that either `.travis.yml` or `.github/workflows/branch.yml` or `.circleci/config.yml` exist

Reply:
Sarek has both GitHub Actions and Travis CI workflows @MaxUlysse ?